.github/workflows/e2e-node.yml

```diff
@@ -12,10 +12,10 @@ jobs:
     if: github.actor != 'dependabot[bot]' && ${{github.event.deployment_status.state == 'success'}}
     steps:
     - uses: actions/checkout@v2.3.4
-    - name: Use Node.js 14.x
+    - name: Use Node.js 16.x
       uses: actions/setup-node@v2.1.5
       with:
-        node-version: 14.x
+        node-version: 16.x
```

Comment:
Should this be setup as a matrix against all supported node versions? 12.x, 14.x, 16.x?

Reply:
Should the e2e tests run across multiple node versions, or is that only required for unit tests ?

Reply:
imo, we only need the matrix on unit tests, which are very likely to catch inconsistencies across node versions. Past that (given that we have 100% unit coverage), there are very quickly diminishing returns.